src/components/panes/CameraTest/CameraTest.tsx

```diff
@@ -9,52 +9,88 @@ import {
   Paper,
   Select,
   Typography,
+  Theme,
+  createStyles,
 } from '@material-ui/core';
 import { ActivePane, useAppStateContext } from '../../AppStateProvider/AppStateProvider';
 import { SmallError } from '../../../icons/SmallError';
 import { useCameraTest } from './useCameraTest/useCameraTest';
 import useDevices from '../../../hooks/useDevices/useDevices';
 
-const useStyles = makeStyles({
-  paper: {
-    padding: '2em',
-    borderRadius: '8px',
-  },
-  videoContainer: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-  },
-  aspectRatioContainer: {
-    position: 'relative',
-    display: 'flex',
-    width: '80%',
-    padding: '1em',
-    margin: '1em 0',
-    '&::after': {
-      content: '""',
-      paddingTop: '56.25%',
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    mainContainer: {
+      display: 'block',
+      [theme.breakpoints.only('md')]: {
+        marginLeft: '3em',
+        width: '70%',
```

Comment:
Instead of this width property here, we could move it to the MuiContainer. This will help the main content block match the width of the nav bar. And for tablet, I think we could try a fixed width of `` `${BREAKPOINTS.md}px` ``